client/layout/masterbar/logged-in.jsx

```diff
@@ -1,4 +1,5 @@
 import config from '@automattic/calypso-config';
+import { isEcommerce } from '@automattic/calypso-products/src';
```

Comment:
What's the difference between `isEcommerce`, `isEcommercePlan` and `isWpComEcommercePlan`?

Seems like `isEcommercePlan` works with a slug only, and that can easily be retrieved from `site.plan?.product_slug`, so do we really need to introduce a new use of the `getSitePlan()` selector?

Reply:
Hey, @tyxla! Thanks for the review :) 

> What's the difference between isEcommerce, isEcommercePlan, and isWpComEcommercePlan?

Great question! It seems that `isEcommerce()` is just a wrapper to `isEcommercePlan()` for grabbing the product slug. If so, then we could definitely simplify this as you suggested! 

Will this `site.plan?.product_slug` be available in all contexts? Would it be better, perhaps, to specifically fetch it using `getSitePlanSlug()`? 

Ps. Sorry for the noobie question; it's my first Calypso PR :)

Reply:
`getSitePlanSlug()` is a great way to get the slug! Let's go with that.

Also congrats on your first PR 😉